prime-dotnet-webapi/Services/interfaces/IHealthAuthoritySiteService.cs

```diff
@@ -11,6 +11,7 @@ public interface IHealthAuthoritySiteService
         Task<bool> SiteExistsAsync(int healthAuthorityId, int siteId);
         Task<HealthAuthoritySiteViewModel> CreateSiteAsync(int healthAuthorityId, HealthAuthoritySiteCreateModel createModel);
         Task<IEnumerable<HealthAuthoritySiteViewModel>> GetSitesAsync(int? healthAuthorityId = null);
```

Comment:
I don't think this method needs `int?` any more, since I assume it was the only way to get all sites before you made the new service method. I'd change it from `int?` => `int` and update the method.